static/app/data/platformCategories.tsx

```diff
@@ -408,6 +408,12 @@ export const feedbackCrashApiPlatforms: readonly PlatformKey[] = [
   'java-log4j2',
   'java-logback',
   'kotlin',
+  'node-awslambda',
+  'node-azurefunctions',
+  'node-connect',
+  'node-gcpfunctions',
+  'node-koa',
+  'node-serverlesscloud',
```

Comment:
I don't think this makes sense for the node serverless platforms, right? 🤔 
So I think only `node-koa` makes sense to add here, the others will usually not have a UI?

Reply:
@michellewzhang you could use the web API banner (#66041) for the others.